packages/ember-metal/lib/core.js

```diff
@@ -94,11 +94,12 @@ if ('undefined' === typeof Ember.ENV.DISABLE_RANGE_API) {
   @static
   @since 1.1.0
 */
+Ember.FEATURES = DEFAULT_FEATURES; //jshint ignore:line
 
-Ember.FEATURES = Ember.ENV.FEATURES;
-
-if (!Ember.FEATURES) {
-  Ember.FEATURES = DEFAULT_FEATURES; //jshint ignore:line
+if (Ember.ENV.FEATURES) {
+  for (var feature in Ember.ENV.FEATURES) {
+    Ember.FEATURES[feature] = Ember.ENV.FEATURES[feature];
```

Comment:
Makes sense. Will update. 

I'm also digging into why the nose tests are failing here (most likely because I am manually manipulating the feature flags there).